package.json

```diff
@@ -26,7 +26,7 @@
     "setup": "npm install && npm run validate -s",
     "validate": "kcd-scripts validate",
     "prepare": "npm run build",
-    "build": "kcd-scripts build",
+    "build": "kcd-scripts build --out-dir lib",
```

Comment:
I have used `lib` folder instead of `dist` 😄

Reply:
Also not necessary, but again, thank you.